src/webpack/patchWebpack.ts

```
@@ -1,353 +1,641 @@
 
-                    mod = lastMod;
-                    code = lastCode;
+                    continue;
                 }
-            }
 
-            if (!patch.all) patches.splice(i--, 1);
-        }
+                code = newCode;
+                patchedSource = `// Webpack Module ${String(moduleId)} - Patched by ${[...patchedBy, patch.plugin].join(", ")}\n${newCode}\n//# sourceURL=WebpackModule${String(moduleId)}`;
+                patchedFactory = (0, eval)(patchedSource);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)